README.rst

```diff
@@ -408,6 +408,8 @@ Development
 
 run *ghcid* with: ``ghcid -c "cabal repl exe:cardano-node --reorder-goals"``
 
+Note: It might be required to set ``ghc-options: -Wwarn`` in `cabal.project` when developing on modules. Otherwise GHC complains about packages specified that are not required.
```

Comment:
It'd be better to recommend setting the existing `development` flags in the `cabal.project` file, e.g.
```
package shelley-spec-ledger
  flags: +development
```
This is what the top level `cardano-haskell` `cabal.project` file does for example.

https://github.com/input-output-hk/cardano-haskell/

Reply:
I will test if this works and change the comment.

Reply:
![image](https://user-images.githubusercontent.com/2385844/120788827-11f0df80-c531-11eb-817e-a350035e3211.png)

I don't think it's sufficient.

Reply:
@dcoutts I've amended the readme to be more specific and added the ``development`` flag as you suggested.